lib/invisible_captcha/view_helpers.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+#frozen_string_literal: true
```

Comment:
🙏🏼  let's try to avoid unnecessary changes please, like this one or some trailing whitespaces

Reply:
Sorry, unintentional